android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java

```diff
@@ -968,6 +968,14 @@ public void requestPermission(Callback callback) {
         requestReadContactsPermission(callback);
     }
 
+    /*
+     * Enable note usage
+     */
+    @ReactMethod
+    public void enableNotesUsage(boolean enabled) {
```

Comment:
could we rename this to `iosEnableNotesUsage`? it would make it obvious you only need to run on ios. Android notes are loaded correctly right?

Reply:
Methods have been renamed. Also, example code has been added as well